getting-started/installation/general-installation/installation-community-containerized.markdown

```diff
@@ -0,0 +1,249 @@
+
+## Overview
+1. Installing container engine
+2. Preparing CFEngine hub in container
+3. Preparing CFEngine host in container
+4. Using docker compose
+    1. Preparing container image for CFEngine
+    2. Using docker compose service
+5. References
+
+## Installing container engine
+**Ref**: [Install Docker Engine](https://docs.docker.com/engine/install/)
+
+OR
+
+**Ref**: [Podman Installation Instructions](https://podman.io/docs/installation)
+(_Optionally_: [Emulating Docker CLI with Podman](https://podman-desktop.io/docs/migrating-from-docker/emulating-docker-cli-with-podman))
+
+## Preparing CFEngine hub in container
+Run the container with systemd
+
+```command
+$ docker run --privileged -dit --name=cfengine-hub registry.access.redhat.com/ubi9-init /usr/sbin/init
+```
+
+Prepare the container for cfengine
+
+```command
+$ docker exec cfengine-hub bash -c "dnf -y update; dnf -y install procps-ng iproute"
+```
+
+Install cfengine-community package
+
+```command
+$ docker exec cfengine-hub bash -c "dnf -y install https://cfengine-package-repos.s3.amazonaws.com/community_binaries/Community-3.24.0/agent_rhel9_x86_64/cfengine-community-3.24.0-1.el9.x86_64.rpm"
+```
+
+Bootstrap cf-agent
+
+```command
+$ docker exec cfengine-hub bash -c "/usr/local/sbin/cf-agent --bootstrap \$(ip -4 -o addr show eth0 | awk '{print \$4}' | cut -d'/' -f1)"
+```
+
+## Preparing CFEngine host in container
+The procedure is similar to the cfengine-hub deployment, with changes to the name of the container for identification.
+
+```command
+$ docker run --privileged -dit --name=cfengine-host registry.access.redhat.com/ubi9-init /usr/sbin/init
+```
+
+Rest of the installation steps remain the same as for **cfengine-hub**.
+
+### Bootstrap cfengine-host to the policy server container.
+Find ip of **cfengine-hub**:
+
+```command
+$ CFENGINE_HUB_IP=$(docker exec cfengine-hub bash -c "ip -4 -o addr show eth0 | awk '{print \$4}' | cut -d'/' -f1")
+```
+
+Bootstrap cfengine-host to cfengine-hub:
+
+```command
+$ docker exec cfengine-host bash -c "/usr/local/sbin/cf-agent --bootstrap ${CFENGINE_HUB_IP}"
+```
+
+## Using docker compose
+### Preparing container image for CFEngine
+Create a `Dockerfile` with following contents:
+
+```cfengine3
+FROM registry.access.redhat.com/ubi9-init:latest
+LABEL container ubi9-init_cfengine-community-3.24.0.1
+
+RUN dnf -y update \
+&& dnf -y install bind-utils iproute procps-ng \
+&& dnf -y install https://cfengine-package-repos.s3.amazonaws.com/community_binaries/Community-3.24.0/agent_rhel9_x86_64/cfengine-community-3.24.0-1.el9.x86_64.rpm
+
+HEALTHCHECK --interval=5s --timeout=15s --retries=3 \
+    CMD /usr/local/sbin/cf-agent --self-diagnostics || exit 1
+
+ENTRYPOINT ["/usr/sbin/init"]
+```
+
+Build the docker image based on above Dockerfile:
+
+```command
+$ docker build -t cfengine:3.24.0-1 -f Dockerfile .
```

Comment:
```suggestion
docker build -t cfengine:3.24.0-1 -f Dockerfile .
```

Reply:
I get a warning.

```
 => WARN: LegacyKeyValueFormat: "LABEL key=value" should be used instead of legacy "LABEL key value" format (line 2)                                                                                                                                                                  0.0s

```